### ISSUE TITLE
Are others model are planned to support in future?

### ISSUE THREAD

**Issue by shivshankar11** [KONAKONA666/q8_kernels#3]
Are others model are planned to support in future?

**Comment by KONAKONA666** [comment]
Yes, I currently working on FLUX/SD3-like models, double + single stream transformers, backward passes.

**Comment by Ratinod** [comment]
I hope [HunyuanVideo](https://github.com/Tencent/HunyuanVideo) will be on the list too. <sub><sup>And I hope it will help reduce VRAM requirements</sup></sub>

**Comment by shivshankar11** [comment]
> > I hope [HunyuanVideo](https://github.com/Tencent/HunyuanVideo) will be on the list too. And I hope it will help reduce VRAM requirements
> 
> i second this, came here to write the same comment 🤣 people are doing a race for who get hunyuan faster the first go go goooo brother
> 
> amazing works btw. thanks

are you same vam content creator?

**Comment by nitinmukesh** [comment]
Yes please support HunyuanVideo
